conf/task_defaults/eurosat.yaml

```diff
@@ -7,7 +7,7 @@ experiment:
     learning_rate_schedule_patience: 6
     weights: "random"
     in_channels: 13
-    num_classes: 10
+    num_classes: 2
```

Comment:
If I'm understanding correctly, this is being changed so the tests will pass. I think we should make this change directly in the test to the config and not in the yaml file. This will confuse users imo because it's not the right num_classes.

Reply:
Correct, see #227. The plan is to move these configs to `tests/conf` and restore the previous config files to `conf`. If you remind me in a few days I can hack on this, should be pretty straightforward.

Reply:
As a consequence of this, torchmetrics throws some insane error messages when you try to train with a config file that doesn't explicitly reset the number of classes in the dataset:

```
RuntimeError: CUDA error: device-side assert triggered
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1.
```

Reply:
I'll open an issue for this so we don't forget